samples/testing/dump_operation/src/Test_SWAP.qs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/// # Sample
+/// Multi File Testing Project
+///
+/// # Description
+/// Organizing code into multiple Q# source files is an important part of
+/// writing readable and maintainable code. In this project, we have `BellState.qs`, `Test_SWAP.qs`,
+/// and `OperationEquivalence.qs`, which contain the operation to be tested.
+/// The presence of a Q# manifest file (`qsharp.json`) tells the compiler
+/// to include all Q# files under `src/`.
+/// These will be tested by Python wrapper, `test_dump_operation.py`
+
+namespace Test_SWAP {
```

Comment:
This file contains the main code that is tested rather than the test code, so I would not call the namespace and the file starting with Test. It's really confusing that it uses Test in the name and the file that has the test logic does not!